content/about/feature-stages/index.md

```diff
@@ -85,24 +84,22 @@ Below is our list of existing features and their current phases. This informatio
 
 | Feature           | Phase
 |-------------------|-------------------
-| [Kubernetes: Envoy Installation and Traffic Interception](/docs/setup/kubernetes/)        | Beta
-| [Kubernetes: Istio Control Plane Installation](/docs/setup/kubernetes/) | Beta
+| [Kubernetes: Envoy Installation and Traffic Interception](/docs/setup/kubernetes/)        | Stable
+| [Kubernetes: Istio Control Plane Installation](/docs/setup/kubernetes/) | Stable
 | [Kubernetes: Istio Control Plane Upgrade](/docs/setup/kubernetes/) | Beta
 | [Pilot Integration into Kubernetes Service Discovery](/docs/setup/kubernetes/)         | Stable
+| [Pilot Integration into Consul](/docs/setup/consul/quick-start/) | Alpha
+| [Pilot Integration into Cloud Foundry Service Discovery](/docs/setup/consul/quick-start/)    | Alpha
```

Comment:
I don't think this link is right. I'm not sure where this should point.

Reply:
You are right, it should point to Cloud Foundry set-up page but we hid that section since it didn't have any instructions other than 'tune into istio-dev for updates' message. I can remove the link if we decide to move forward with this PR.